reana_db/cli.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA DB command line."""
+
+import os
+
+from alembic import command
+from alembic import config as alembic_config
+import click
+
+from reana_db.database import init_db
+
+
+@click.group()
+def cli():
+    """REANA database commands."""
+
+
+@cli.command()
+def init():
+    """Show REANA database migration recipes history."""
+    init_db()
+    click.secho("Database initialised.", fg="green")
+
+
+@cli.group("alembic")
+@click.pass_context
+def alembic_group(ctx):
+    """REANA database migration commands.
+
+    Note that this command is just a light wrapper around alembic.
+    """
+    reana_alembic_ini = os.path.join(os.path.dirname(__file__), "alembic.ini")
+    config = alembic_config.Config(reana_alembic_ini)
+    ctx.obj = config
+
+
+@alembic_group.command()
+@click.option(
+    "-m", "--message", default=None, help="Message string to use with 'revision'"
+)
+@click.option(
+    "--autogenerate",
```

Comment:
I was checking this option as I think it can be pretty useful to generate prefilled revision but I get
```console
alembic.util.exc.CommandError: Can't proceed with --autogenerate option; environment script /code/modules/reana-db/reana_db/alembic/env.py does not provide a MetaData object or sequence of objects to the context.
```

I think we need to add MetaData to all the models as stated [here](https://alembic.sqlalchemy.org/en/latest/api/autogenerate.html#getting-diffs)

Reply:
If that enables autogenerating migrations let's do it, [here](https://alembic.sqlalchemy.org/en/latest/autogenerate.html) the specific implementation details.

Reply:
I wonder if this option should be enabled by default, WDYT?

The generated result should look something like this, so one needs to adjust it in any case:

```python
def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.add_column("user_", sa.Column("test", sa.String(length=255), nullable=True))
    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_column("user_", "test")
    # ### end Alembic commands ###
```